curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations/css-foundations-question-g.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 63ee35450d8d4841c3a70920
+videoId: LGQuIIv2RVA
+title: CSS Foundations Question G
+challengeType: 15
+dashedName: css-foundations-question-g
+---
+# --description--
+
+Internal CSS (or embedded CSS) involves adding the CSS within the HTML file itself instead of creating a completely separate file. With the internal method, you place all the rules inside of a pair of opening and closing `<style>` tags, which are then placed inside of the opening and closing `<head>` tags of your HTML file. Since the styles are being placed directly inside of the `<head>` tags, we no longer need a `<link>` element that the external method requires.
```

Comment:
```suggestion
Internal CSS (or embedded CSS) involves adding the CSS within the HTML file itself instead of creating a completely separate file. With the internal method, you place all the rules inside of a pair of opening and closing `<style>` tags, which are then placed inside of the opening and closing `<head>` tags of your HTML file. Since the styles are being placed directly inside of the `<head>` tags, you no longer need a `<link>` element that the external method requires.
```